catalystneuro/nwb-conversion-tools
# added spikegadgets data interface with new source_schema setting method

## Motivation
SpikeGadgetsRawIO has been merged to neo master ([PR #954](https://github.com/NeuralEnsemble/python-neo/pull/954)), and SpikeGadgetsRecordingExtractor has been merged to spikeextractors master ([PR #592](https://github.com/SpikeInterface/spikeextractors/pull/592)).

This is the next step of building the `SpikeGadgetsDataInterface`. Pretty simple, doesn't require a lot of extra setting in terms of metadata, aside from the optional probe file.

## How to test the behavior?
Existing schema tests should cover the new interface since it was added to the `interface_list`.

Other than that, a big to-do for this and all the other data interfaces is to add GIN and other running tests to ensure they function as expected.

## Checklist

- [X] Is your contribution compliant with our coding style? Read about [PEP8](https://www.python.org/dev/peps/pep-0008/) and [black](https://black.readthedocs.io/en/stable/the_black_code_style.html).
